admin_guide/managing_projects.adoc

```diff
@@ -376,3 +376,48 @@ ifdef::openshift-enterprise[]
 # systemctl restart atomic-openshift-master-api atomic-openshift-master-controllers
 ----
 endif::[]
+
+ifdef::openshift-dedicated[]
+[[enforcing-different-limits-on-users-projects]]
+== Enforcing Different Limits on Users' Projects
```

Comment:
The title is not clear - I would suggest something like:
`Enforcing Different Project Limits on/for Users`